### ISSUE TITLE
--env-file flag only works with path relative to the container-compose filepath

### ISSUE THREAD

**Issue by SiRumCz** [containers/podman-compose#636]
**Describe the bug**

--env-file flag only works with path relative to the container-compose filepath.

please try to reproduce the bug in latest devel branch

**To Reproduce**
here's my project layout:
```
root/
    - proj1/
        - container-compose.yml
        - envs/
            - 1.env
```
1. if run from the root `podman-compose -f proj1/container-compose.yml --env-file proj1/envs/1.env`, the env vars will not be loaded.

**Expected behavior**
The env vars should be loaded when env-file points to the env file path.

**Actual behavior**
What is the behavior you actually got and that should not happen.

env vars not loaded.

**Environment:**
 - OS: WSL
 - podman version: 4.4.1
 - podman compose version: 1.0.4(built from latest)

**Additional context**

1. if run from the root `podman-compose -f proj1/container-compose.yml --env-file envs/1.env`, the env vars will be loaded correctly.

Please correct me if this is the expected behavior for podman/podman-compose.

**Comment by bugfest** [comment]
Hi @SiRumCz, 

This behavior is due to https://github.com/containers/podman-compose/blob/b8a7593026002def300d775b5114f476b4b47e08/podman_compose.py#L1440 as `podman-compose` changes the work directory to the referenced compose file is no longer able to find the env file using a relative path.

**Comment by bugfest** [comment]
Fix proposal submitted (PR #656)

**Comment by muayyad-alsadi** [comment]
it's merged, please verify and close
